_commands/tools/ping.md

```diff
@@ -0,0 +1,72 @@
+---
+---
+
+ping
+--
+
+`ping` is used to test connections to other machines on a network.
+
+~~~ bash
+$ ping 192.168.1.1
+PING 192.168.1.1 (192.168.1.1): 56 data bytes
+64 bytes from 192.168.1.1: icmp_seq=0 ttl=255 time=1.518 ms
+$
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+
+#### `ping -i`
+
+~~~ bash
+$ ping -i 1 192.168.1.1
```

Comment:
Switch one of these to `ping www.google.com` or something like that, and show how it can be quickly used to test internet connectivity.